File: solutions/168.excel-sheet-column-title/excel-sheet-column-title.js
/**
 * @param {number} n
 * @return {string}
 */
var convertToTitle = function(n) {

    let res = '';
    while(n > 0){
        res = String.fromCharCode(65 + (n - 1) % 26 ) + res;
        n = parseInt((n - 1) / 26);
    }
    return res;
    
};

File: solutions/168.excel-sheet-column-title/excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int n) {
        String res = "";
        while(n != 0){
            n = n -1;
            res = (char)(n % 26 + 'A') + res;
            n = n /26;
        }
        return res;
    }
}